cloudformation/batch-substack.cfn.json

```diff
@@ -121,8 +121,29 @@
           ]
         },
         "ManagedPolicyArns": [
-          "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess",
+          {
+            "Fn::Sub": "arn:${AWS::Partition}:iam::aws:policy/AmazonS3ReadOnlyAccess"
+          },
           "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
```

Comment:
I know it's not part of your PR but I was just wondering if we should also use {AWS::Partition} here

Reply:
fixed